axelrod/strategies/finite_state_machines.py

```diff
@@ -101,14 +123,15 @@ class Fortress3(FSMPlayer):
     }
 
     def __init__(self) -> None:
-        transitions = (
-            (1, D, 2, D),
+        transitions = [
```

Comment:
the FSM  and FSMPlayer used a list.  then it got changed to tuple in all the subsequent ones.  i spent about 15 minutes scratching my head trying to figure out why mypy was complaining before i noticed the difference.  i used list because that's what i saw first and they all needed to be the same.  

shall i change all to tuples or leave as is?